Tests/scripts/validate.sh

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env bash
 set -e
 
-echo "CIRCLE_BRANCH: $CIRCLE_BRANCH CI: $CI DEMISTO_README_VALIDATION: $DEMISTO_README_VALIDATION"
+echo "CI_COMMIT_BRANCH: $CI_COMMIT_BRANCH CI: $CI DEMISTO_README_VALIDATION: $DEMISTO_README_VALIDATION"
 
-if [[ $CIRCLE_BRANCH = master ]] || [[ -n "${NIGHTLY}" ]] || [[ -n "${BUCKET_UPLOAD}" ]] || [[ -n "${DEMISTO_SDK_NIGHTLY}" ]]; then
-    demisto-sdk validate -a --post-commit --id-set --id-set-path $CIRCLE_ARTIFACTS/unified_id_set.json
-elif [[ $CIRCLE_BRANCH =~ pull/[0-9]+ ]]; then
-    demisto-sdk validate -g --post-commit --id-set --id-set-path ./Tests/id_set.json
+if [[ $CI_COMMIT_BRANCH = master ]] || [[ -n "${NIGHTLY}" ]] || [[ -n "${BUCKET_UPLOAD}" ]] || [[ -n "${DEMISTO_SDK_NIGHTLY}" ]]; then
+    demisto-sdk validate -a --post-commit --id-set --id-set-path "$ARTIFACTS_FOLDER/unified_id_set.json"
+elif [[ $CI_COMMIT_BRANCH =~ pull/[0-9]+ ]]; then
+    demisto-sdk validate -g --post-commit --id-set --id-set-path "$ARTIFACTS_FOLDER/id_set.json"
```

Comment:
```suggestion
if [[ $CI_COMMIT_BRANCH =~ pull/[0-9]+ ]]; then
    demisto-sdk validate -g --post-commit --id-set --id-set-path "$ARTIFACTS_FOLDER/id_set.json"
else
    demisto-sdk validate -g --post-commit --id-set --id-set-path "$ARTIFACTS_FOLDER/unified_id_set.json"
fi
```
Seems like the only difference here is whether it's a contribution branch or not.

Reply:
The options are different. One is run with the `-g` flag, the other with `-a`.